src/App.jsx

```diff
@@ -1,7 +1,88 @@
 import "./App.css";
+import { Container, Row, Col } from "react-bootstrap";
```

Comment:
Sitaip kai importini, importinasi visas bootstrap component library, siulau importini po viena elementa kuri naudoji kaip :

import Container from "react-bootstrap/Container";
import Row from "react-bootstrap/Row";
import Col from "react-bootstrap/Col";

Net pats developeris taip rekomenduoja :
"You should import individual components like: react-bootstrap/Button rather than the entire library. Doing so pulls in only the specific components that you use, which can significantly reduce the amount of code you end up sending to the client."  -- https://react-bootstrap.netlify.app/docs/getting-started/introduction

Reply:
Pataisyta